cache-config/t3c-diff/t3c-diff.go

```diff
@@ -36,31 +36,31 @@ func main() {
 	help := getopt.BoolLong("help", 'h', "Print usage info and exit")
 	getopt.ParseV2()
 	if *help {
-		fmt.Println(usageStr)
+		log.Errorln(usageStr)
```

Comment:
`t3c-diff` hasn't initialized the loggers, needs a `log.Init(os.Stderr, os.Stderr, os.Stderr, os.Stderr, os.Stderr)`

Reply:
Added log.Init()